Dockerfile

```diff
@@ -1,5 +1,21 @@
 FROM node
```

Comment:
There is a problem in this file, probably a conflict which is not merged. You should resolve the conflict and merge the result before continue; VS Code should help with this.